src/main/java/me/totalfreedom/totalfreedommod/rank/Title.java

```diff
@@ -7,6 +7,7 @@
 {
 
     DEVELOPER("a", "Developer", ChatColor.DARK_PURPLE, "Dev"),
+    FOUNDER("the", "Founder", ChatColor.DARK_RED, "TF Founder"),
```

Comment:
It would make it inaccurate on other servers using TFM. It should be generic to be used on all servers, or a system put in place to check if this plugin is running on TF or not, and only display that rank on TF.